### DISCUSSION TITLE
is it possible to enable "open link in a new tab" throughout the web-app?

### DISCUSSION THREAD

**Discussion by asaf-b** [keygen-sh/community#62] [Ideas]
Hi Zeke,
When dealing with customer support, and in general, I often need to make manual actions via the web-app.
Since I'm dealing with multiple licenses/machines per user, I find the fact that it isn't possible to "open link in a new tab" a bit annoying.
For example- I need to update the expiry date of 5 licenses for a user. So I use the search to find that user object very easily, then go into its related licenses. Once I see all licenses, I need to go into each one and edit it. Now that the "open link in a new tab" isn't possible I need to go into each license, edit it, then find my way back to the licenses list by using the "back" button, re-searching, or pasting the link I copied into the URL bar.
If I was able to open the licenses in a new tab, without losing the licenses page- that would be much easier to deal with.
That is true to all object links within the web-app for that matter.
Is it possible to add this ability throughout?

Thank you,
Asaf Ben Zur

**Comment by ezekg** [comment]
I agree that this is frustrating. I'll try and put in some time this week to resolve this issue.
